content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Spectro Cloud Grafana Dashboards'
+metaTitle: 'Spectro Cloud Grafana Dashboards'
+metaDescription: 'Learn more about the Spectro Cloud Grafana dashboard and how to use them.'
```

Comment:
Is it called "Spectro Cloud Grafana Dashboard"? or is it "Spectro Grafana Dashboard"? (I'm thinking about consistency with other Spectro Dashboards, but this one is new to me.)

Should "them" be "it"?

Line 17 below has plural Dashboards, so maybe this is a dashboard that consists of several components?